### ISSUE TITLE
Poor integration with huggingface transformers

### ISSUE THREAD

**Issue by C-J-Cundy** [C-J-Cundy/seq-viz#4]
We should have better integration with the transformers libary. In particular, we should not require recomputing logits. We should be able to use a function that can be a plug-in replacement for `compute_metrics` with the side effect of saving a file that can be used for the sequence visualizer. This would minimize the overhead that a user has to incur when using the sequence visualizer with the default SFTTrainer with TRL, for instance.

**Comment by C-J-Cundy** [comment]
Fixed by #6
